src/Http/Http.Extensions/test/RequestDelegateGenerator/CompileTimeCreationTests.cs

```diff
@@ -96,6 +97,43 @@ public async Task SupportsDifferentInterceptorsFromSameLocation()
         await VerifyAgainstBaselineUsingFile(updatedCompilation);
     }
 
+    [Fact]
+    public async Task SupportsMapCallOnNewLine()
+    {
+        var source = """
+app
+     .MapGet("/hello1/{id}", (int id) => $"Hello {id}!");
+EndpointRouteBuilderExtensions
+    .MapGet(app, "/hello2/{id}", (int id) => $"Hello {id}!");
+app.
+    MapGet("/hello1/{id}", (int id) => $"Hello {id}!");
+EndpointRouteBuilderExtensions.
+   MapGet(app, "/hello2/{id}", (int id) => $"Hello {id}!");
+app.
+MapGet("/hello1/{id}", (int id) => $"Hello {id}!");
+EndpointRouteBuilderExtensions.
+MapGet(app, "/hello2/{id}", (int id) => $"Hello {id}!");
+app.
+
+
+MapGet("/hello1/{id}", (int id) => $"Hello {id}!");
+EndpointRouteBuilderExtensions.
+
+
+MapGet(app, "/hello2/{id}", (int id) => $"Hello {id}!");
```

Comment:
Is
```
app.
MapGet
(...);
```
interesting enough to add a test for?

Reply:
Doesn't hurt! Added a few more scenarios including the operator token being on a separate line.